docs/src/autotuning-ridge.md

```diff
@@ -0,0 +1,211 @@
+```@example 3
+using DiffOpt
+using Statistics
+using OSQP
+using JuMP
+using Plots
+import Random
+using LinearAlgebra
+using Printf
+Base.show(io::IO, f::Float64) = @printf(io, "%1.3f", f) # to reduce float precision while printing
+nothing # hide
+```
+
+
+```@example 3
+"""
+    Return the coefficient of determination R2 of the prediction.
+    best possible score is 1.0
+    it can be negative (because the model can be arbitrarily worse)
```

Comment:
```suggestion
Return the coefficient of determination R2 of the prediction.
Best possible score is 1.0, it can be negative because the model can be arbitrarily worse
```